edb/server/args.py

```diff
@@ -372,6 +377,13 @@ def parse_args(**kwargs: Any):
                 DeprecationWarning,
             )
 
+    if kwargs['auto_shutdown']:
```

Comment:
Let's turn this into `auto_shutdown_after = 0` and remove the handling of `auto_shutdown`.

Reply:
changed